workers/unity/Packages/io.improbable.gdk.core/Representation/Types/SimpleEntityResolver.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core.Representation.Types
+{
+    [Serializable]
+    public class SimpleEntityResolver : IEntityRepresentationResolver
+    {
+        public SimpleEntityResolver()
+        {
+        }
+
+        public SimpleEntityResolver(string entityType, GameObject prefab)
```

Comment:
Why do some have constructors and some don't?

Reply:
I asked this same question and I thinks that this one is used in tests so you need a way to instantiate it.